Src/Plugins/Security/Security.SharedStrings.txt

```diff
@@ -96,7 +96,7 @@
   $SEC101/027.MailChimpApiKey=(?:[^0-9a-f]|^)(?P<secret>[0-9a-f]{32}-us[0-9]{12})
   $SEC101/029.AlibabaCloudAccessKey=(?:^|[^0-9A-Za-z])(?P<id>LTAI([0-9A-Za-z]{12}|[0-9A-Za-z]{20}))(?:[^0-9A-Za-z]|$)
   $SEC101/029.AlibabaCloudSecretKey=(?:^|[^0-9A-Za-z\-~_+/%\\])(?P<secret>[0-9A-Za-z]{30})(?:[^0-9A-Za-z\-~_+/=%\\]|$)
-  $SEC101/033.MongoDbCredentials=(?i)mongodb(?:[^:]{0,50})?:\/\/(?P<id>[0-9a-z]+):(?P<secret>[^@\s]{1,200})@(?P<host>[^\/;"<\s,]+)
+  $SEC101/033.MongoDbCredentials=(?i)mongodb(?P<schema>\+srv)?:\/\/(?P<id>[0-9a-z_-]+):(?P<secret>[^@\s]{1,200})@(?P<host>[^\/;"<\s,]+)(?P<options>/[0-9a-z?&@=_-]+)?
```

Comment:
>schema [](http://example.com/codeflow?start=48&length=6)

why is this named 'schema' and not 'scheme' (if it even requires us to have a named group.

Reply:
thinking about capturing the whole protocol part using `(?P<protocol>mongodb|mongodb\+srv)` instead, will it be more clear?